cmd/skopeo/signing.go

```diff
@@ -98,17 +104,45 @@ func (opts *standaloneVerifyOptions) run(args []string, stdout io.Writer) error
 		return fmt.Errorf("Error reading signature from %s: %v", signaturePath, err)
 	}
 
-	mech, err := signature.NewGPGSigningMechanism()
+	var mech signature.SigningMechanism
+	var fingerprints []string
+	if opts.publickeyfile != "" {
+		publicKeys, err := os.ReadFile(opts.publickeyfile)
+		if err != nil {
+			return fmt.Errorf("Error reading public keys from %s: %v", opts.publickeyfile, err)
```

Comment:
(Absolutely non-blocking: Using `%w` would be more idiomatic — but it doesn’t make a difference in practice, and there’s quite a lot of pre-existing `%v`.)